tests/dotnet/MyClassRedirectApp/AppDelegate.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Runtime.InteropServices;
+using Foundation;
+using ObjCRuntime;
+
+namespace MySimpleApp {
+	public class Program {
+		static int Main (string [] args)
+		{
+			var someObj = new SomeObj ();
+			var handle = someObj.ClassHandle;
+			if (handle == NativeHandle.Zero)
+				return 1;
+			return 0;
```

Comment:
```suggestion
			return someObj.ClassHandle == NativeHandle.Zero ? 1 : 0;
```
Small suggestion for conciseness